front/chat/src/Components/Chat.svelte

```diff
@@ -132,7 +132,13 @@
         {#if $connectionNotAuthorized}
             <NeedRefresh />
         {:else if !connectionManager.connection || !$xmppServerConnectionStatusStore}
-            <Loader text={$userStore ? $LL.reconnecting() : $LL.waitingData()} />
+            <Loader
+                text={$userStore
+                    ? !connectionManager.connection
```

Comment:
Is this supposed to work? `connectionManager.connection` is not reactive, so it should not as far as I understand, it should not be updated

Reply:
You re right, just updated that